### PR TITLE
added tasks objects

### PR DESCRIPTION
**Pull request by YuvHayun**
## Related Issues
fixes: [CIAC-10864](https://jira-dc.paloaltonetworks.com/browse/CIAC-10864)

## Description
Modelized playbook tasks into objects.

### CODE REVIEW

File: .changelog/4343.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Mobilized base_playbook tasks into objects to ease on the use in various flows.
```
Comment (JudahSchwartz) [inline .changelog/4343.yml:2]: mobilized is definitely not the right word

Comment (YuvHayun) [reply .changelog/4343.yml:2]: Meant modelized

File: demisto_sdk/commands/content_graph/objects/base_playbook.py [hunk 2]
```diff
@@ -48,6 +204,15 @@ def prepare_for_upload(
     def _client_upload_method(cls, client: demisto_client) -> Callable:
         return client.import_playbook
 
+    def save(self):
+        super().save()
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/content_graph/objects/base_playbook.py:208]: shouldnt call save after?

**Review by JudahSchwartz — APPROVED** [review]

File: demisto_sdk/commands/content_graph/objects/base_playbook.py (hunk 2 above)
Comment (YuvHayun) [reply demisto_sdk/commands/content_graph/objects/base_playbook.py:208]: Nope, we want to override the dict once we're done.

**Pull request merged**